### DISCUSSION TITLE
5.0 版本中是否支持在边上绘制粒子效果

### DISCUSSION THREAD

**Discussion by AlichChen** [antvis/G6#5698] [Q&A]
如题： 5.0 版本中是否支持在边上绘制粒子效果

在G示例中有 CanvasKit  的效果，4.X 版本中没找到可支持 CanvasKit  的能力，请问 5.0 中是否有计划？

![image](https://github.com/antvis/G6/assets/7791862/f96f6f59-90db-43aa-a51b-f08dfe039e8e)

**Comment by Aarebecca** [comment]
目前还不支持这个能力
